src/sas/qtgui/Perspectives/Inversion/InversionPerspective.py

```diff
@@ -111,21 +85,83 @@ def __init__(self, parent=None, data=None):
         self.isBatch = False
         self.batchResultsWindow = None
         self.batchResults = {}
-        self.batchComplete = []
-
-        # Add validators
-        self.setupValidators()
-        # Link user interactions with methods
-        self.setupLinks()
-        # Set values
-        self.setupModel()
-        # Set up the Widget Map
-        self.setupMapper()
-
-        #Hidding calculate all buton
-        self.calculateAllButton.setVisible(False)
-        # Set base window state
-        self.setupWindow()
+
+        # Max index for adding new, non-clashing tab names
+        self.maxIndex = 1
+
+        # The tabs need to be closeable
+        self.setTabsClosable(True)
+
+        # The tabs need to be movable
+        self.setMovable(True)
+
+        # Initialize the first tab
+        self.addData(None)
+
+    ######################################################################
+    # Batch Mode and Tab Functions
+
+    def calculate(self):
```

Comment:
I don't see any usages of this method.

Reply:
I've removed that function since it was something I was trying out but didn't end up using at the end. Sorry about that it wasn't supposed to end up in the final version.